tests/Avalonia.UnitTests/MockPlatformRenderInterface.cs

```diff
@@ -9,6 +9,8 @@ namespace Avalonia.UnitTests
 {
     public class MockPlatformRenderInterface : IPlatformRenderInterface
     {
+        public IEnumerable<string> InstalledFontNames => Mock.Of<IEnumerable<string>>();
```

Comment:
Probably better to return `new string[0]` from both mocks

Reply:
There you go